client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/useInvestigatedPersonInfo.ts

```diff
@@ -27,14 +27,19 @@ const useInvestigatedPersonInfo = (): InvestigatedPersonInfoOutcome => {
     const userRole = useSelector<StoreStateType, number>(state => state.user.data.userType);
     const currInvestigatorId = useSelector<StoreStateType, string>(state => state.investigation.creator);
     const investigationStatus = useSelector<StoreStateType, InvestigationStatus>(state => state.investigation.investigationStatus);
-
+    
     const handleInvestigationFinish = async () => {
         alertSuccess('בחרת לצאת מהחקירה לפני השלמתה! הנך מועבר לעמוד הנחיתה', {
             timer: 1750,
             showConfirmButton: false
         });
         timeout(1500).then(() => {
-            setIsInInvestigation(false);
+            const bc = new BroadcastChannel(BC_TABS_NAME);
+            const broadcastMessage : BroadcastMessage = {
```

Comment:
use meaningful name -  cloasingInvastigationBroadcastMassage for example

Reply:
relevant for all usage in bc variable

Reply:
DONE